configs/default/AFNG-ALIENFLIGHTNGF7.config

```
@@ -1,8 +1,13 @@
-# Betaflight / STM32F7X2 (S7X2) 4.1.0 Oct 16 2019 / 11:58:45 (c37a7c91a) MSP API: 1.42
+# Betaflight / STM32F7X2 (S7X2) 4.3.2 Nov 28 2022 / 07:30:19 (60c9521) MSP API: 1.44
 
 #define USE_GYRO_SPI_MPU6500
 #define USE_ACC_SPI_MPU6500
+#define USE_GYRO_SPI_ICM20602
+#define USE_ACC_SPI_ICM20602
```

ICM20602 is handled by MPU6500

```suggestion
#define USE_GYRO_SPI_MPU6500
#define USE_ACC_SPI_MPU6500
```

---

Aren't they independent gyros?

---

There are a few different AFNG boards using different hardware but same firmware. Official board designs are published here: https://www.alienflightng.com/
Anyhow there are some more boards out in the wild wich are based on the AFNG design. They are build by AFNG team members and in the AF community. These are boards with MPU6000 or MPU9250. These are unpublished porotypes which likely can be handled by custom defines in the configurator.

---

@KarateBrot these gyro's are not much used anymore.
If you look in gyro code you see the `MPU6500` handles `ICM2060x` also:

https://github.com/betaflight/betaflight/blob/a3a21d9233660d5f9bcd47d62f3a550e27b04e6f/src/main/sensors/gyro_init.c#L402-L431

---

Okay I see. I still think it makes sense to include the correct one for completeness. Just `#define` whatever is on the board and let the pre-compiler do its job afterwards.

---

@KarateBrot Do you want me to make changes in firmware to make this happen?

---

It already does that in firmware, that's my point. We don't need to change the define to MPU6500.

---

https://github.com/betaflight/betaflight/pull/12161

---

@MJ666 Thanks to haslinghuis you can now `#define` the actual gyro that is on board (ICM20602). But you can also leave it as MPU6500. The board config will still work. I prefer to have the actual gyro name in the board config but it's your call 👍

---

Will make a new PR to update and add all sensors used on different AFNG boards.